lib/_tools.scss

```diff
@@ -76,14 +75,10 @@
         -webkit-flex-flow: row wrap;
         -ms-flex-flow: row wrap;
         flex-flow: row wrap;
-        -webkit-flex: 1 1 auto; // [2]
-        -ms-flex: 1 1 auto; // [2]
-        flex: 1 1 auto; // [2]
-
-        margin-right: 0; // [3]
+        margin-right: 0; // [2]
```

Comment:
It's not needed here. Because the grid is primarily built with flexbox if you set `direction="rtl" then it automatically flips the grid and then we already have `flip` in place for the `inline-block` fallback here: https://github.com/bbc/gel-grid/blob/master/lib/_tools.scss#L69-L70